src/main/java/dev/mlml/command/CommandRegistry.java

```diff
@@ -92,7 +93,11 @@ public static void executeCommand(Message message) {
 
         long cooldown = getCooldown(command, ctx.getMember().getId());
         if (cooldown > 0) {
-            message.reply(String.format("You must wait %d seconds before using this command again", cooldown / 1000))
+            String prettyPrint = Utils.timePrettyPrint(cooldown);
+
+            String cooldownReply = String.format("You must wait %s before using this command again!", prettyPrint);
```

Comment:
This would be better as one line, the prettyPrint variable doesn't add readability as there's only one placeholder in this format call and the call to `timePrettyPrint` is really simple